packages/fast-components-react-base/src/breadcrumb-item/breadcrumb-item.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import { get } from "lodash-es";
+import Hypertext, { HypertextClassNameContract } from "../hypertext";
+import { BreadcrumbItemClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+import {
+    BreadcrumbItemHandledProps,
+    BreadcrumbItemProps,
+    BreadcrumbItemUnhandledProps,
+} from "./breadcrumb-item.props";
+
+class BreadcrumbItem extends Foundation<
+    BreadcrumbItemHandledProps,
+    BreadcrumbItemUnhandledProps,
+    {}
+> {
+    public static displayName: string = "BreadcrumbItem";
+
+    public static defaultProps: Partial<BreadcrumbItemHandledProps> = {
+        current: false,

```

Comment:
nit: remove comma

Reply:
Prettier will catch this if it's important; this may be added by prettier as well (ES5 trailing comma)